views/peeps/index.erb

```diff
@@ -0,0 +1,5 @@
+<ul>
+  <% @peeps.each do |peeps| %>
+  <li><%= peeps %></li>
```

Comment:
Lovely! I'd be tempted to call the inner variable `peep` rather than `peeps` to represent the fact that each item in the list is a singular peep for readability.